src/services/releases.js

```diff
@@ -24,7 +24,7 @@ const getRevisions = (name, product) => {
           product,
           data_version: Number(dataVersion) ? dataVersion : parts[0],
           timestamp: parseInt(parts[1], 10),
-          changed_by: parts[2],
+          changed_by: parts.slice(2).join('-'),
```

Comment:
Out of curiosity, why do `parts.slice(2)` instead of `parts[2]`?

Reply:
For usernames like `balrog-ffxbld`, we end up with more than 3 items in the Array, so we want to grab the 3rd item and on:
```
var a = "Devedition-68.0b1-build1/483-1557763308625-balrog-ffxbld.json"
undefined
a.replace(`Devedition-68.0b1-build1/`, '').replace('.json', '').split('-')[2]
"balrog"
a.replace(`Devedition-68.0b1-build1/`, '').replace('.json', '').split('-').slice(2).join("-")
"balrog-ffxbld"

var a = "Devedition-68.0b1-build1/483-1557763308625-bhearsum@mozilla.com.json"
undefined
a.replace(`Devedition-68.0b1-build1/`, '').replace('.json', '').split('-')
Array(3) [ "483", "1557763308625", "bhearsum@mozilla.com" ]
a.replace(`Devedition-68.0b1-build1/`, '').replace('.json', '').split('-').slice(2).join("-")
"bhearsum@mozilla.com"
```

Reply:
Got it, thanks!